packages/css/utils.css

```diff
@@ -24,3 +24,19 @@
     box-shadow: 0 0 0 var(--fds-focus-border-width) var(--fds-semantic-border-focus-boxshadow);
   }
 }
+
+@layer fds.utils.animate-height {
+  .fds-animate-height--openingOrClosing,
+  .fds-animate-height--closed {
+    overflow: hidden;
+  }
+
+  .fds-animate-height--open .fds-animate-height__content {
+    height: auto;
+  }
+
+  .fds-animate-height--closed .fds-animate-height__content {
+    height: 0;
+    display: none;
+  }
+}
```

Comment:
I believe you can just make a new layer `animate-height` under `fds.utils`, and it nests correctly:

```css
@layer fds.utils {
  ...

  @layer animate-height {
    ...
  }
}
```

Reply:
Jepp 
![image](https://github.com/digdir/designsystemet/assets/1070981/5971f1fb-2d57-4f6d-9662-340141e69604)